app/controllers/lunches/details/contributors.js

```diff
@@ -0,0 +1,47 @@
+var loader,
+    LoaderInstance;
+
+/**
+ *  Constructor
+ */
+( function constructor( args ) {
+
+    initializeLoader();
+
+} )( arguments[ 0 ] || {} );
+
+
+function initializeLoader() {
+    LoaderInstance = require( "/loader" );
+    loader = new LoaderInstance( $.window );
+}
+
+function populateContributors() {
+    var api = require( "/api" );
+    loader.show();
+
+    var items = [];
+    api.getContrib( {
+
+    }, function( payload ) {
```

Comment:
Leave out the 1st parameter if there are no params. Adjst the api method accordingly.